homeassistant/components/smtp/notify.py

```diff
@@ -255,13 +256,25 @@ def _attach_file(hass, atch_name, content_id=""):
     """
     try:
         file_path = Path(atch_name).parent
-        if not hass.config.is_allowed_path(str(file_path)):
-            _LOGGER.warning(
-                "'%s' is not secure to load data from, ignoring attachment '%s'!",
-                file_path,
-                atch_name,
+        if os.path.exists(file_path) and not hass.config.is_allowed_path(
+            str(file_path)
+        ):
+            file_name = os.path.basename(atch_name)
+            url = "https://www.home-assistant.io/docs/configuration/basic/"
+            raise ServiceValidationError(
+                f"Cannot send email with attachment '{file_name} "
+                f"from directory '{file_path} which is not secure to load data from. "
+                "Only folders added to `allowlist_external_dirs` are accessible. See "
```

Comment:
I'm wondering if `allowlist_external_dirs` should also be a placeholder. That option isn't translated, right?

Reply:
It is not translated, but even could be empty. I am not sure we should add this as a placeholder.

Reply:
What do you mean? It's just a string for the option name.

Reply:
Ah, I thought you ment passing the content of the setting.